polaris-react/src/components/Badge/Badge.scss

```diff
@@ -14,7 +14,7 @@
   color: var(--p-color-text);
 
   #{$se23} & {
-    background-color: var(--p-color-experimental-bg-transparent-subdued);
+    background-color: var(--p-color-bg-transparent-subdued-experimental);
```

Comment:
It looks like the experimental color changes weren't pushed up to this branch, or just missed?

Reply:
Whoops, yup tab was unsaved and never pushed up! Updated https://github.com/Shopify/polaris/pull/9349/commits/9aac2a483f07a92d556f85ff2b4adb1df32d6a3e, https://github.com/Shopify/polaris/pull/9349/commits/38a8c32a7b2bf9e8728f151b999833b7c135b6f5